src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -824,7 +828,39 @@ function RemoteFunctions(experimental) {
     if (experimental) {
         window.document.addEventListener("keydown", onKeyDown);
     }
+    
+    var _ws = null;
+
+    function onDocumentClick(event) {
+        var element = event.target,
+            currentDataId,
+            newDataId;
+        if (element && element.hasAttribute('data-brackets-id')) {
+            _ws.send(JSON.stringify({
+                type: "message",
+                message: element.getAttribute('data-brackets-id')
```

Comment:
The message is just the element id, correct? When sending messages in other direction (from Brackets to browser), the message has the format:

```
{ method: method, id: id, params: params }
```

It would be nice to do the same here so that's it's easier to add any new messages in the future.